.ci/travis/travis.sh

```diff
@@ -655,6 +655,62 @@ check-since-version)
   fi
   ;;
 
+checkstyle-cli-run-openjdk14)
+  # Set up environment
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  CHECKSTYLE_DIR=$(pwd)
+  TRAVIS_DIR="${CHECKSTYLE_DIR}/.ci/travis"
+  CHECKSTYLE_CONFIG="${CHECKSTYLE_DIR}/.ci/travis/openjdk14-test/single-module-config.xml"
+  FILTER_FILE="${CHECKSTYLE_DIR}/.ci/travis/openjdk14-test/jdk14-test-excluded-files.list"
+  INPUT_FILE="$(mktemp)"
```

Comment:
This variable is not used and is blindly overwritten in line 680. Please remove.